.github/workflows/pull_request_label.yml

```diff
@@ -0,0 +1,20 @@
+name: PR label
+
+on:
+  pull_request:
+    types: [labeled, unlabeled, opened, reopened, synchronize]
+
+jobs:
+  semver-label-check:
+    name: Semantic Version label check
+    runs-on: ubuntu-latest
+    timeout-minutes: 1
+    steps:
+      - name: Checkout repository
+        if: ${{ !env.ACT }}
```

Comment:
Should we move that into the semantic version action itself so we don't have to spread this if check everywhere?

Reply:
👍 removed for now and I'll open a PR in NIO

Reply:
https://github.com/apple/swift-nio/pull/2926